### PR TITLE
vpx-flintstones

### PR DESCRIPTION
**Pull request by Bla1ze**
Please ensure your PR includes the following:

- README.md
- launcher.png
- launcher.xml
- pinmame/cfg, pinmame/ini, pinmame/nvram, pinmame/roms folders. If they are empty, create a .gitkeep file to ensure the folders get created in the repo.
- any game specific ini or vbs files named exactly the same as the publicly available files from either VPForums or VPUniverse

Please ensure your PR **DOES NOT** include any of the following:

- VPX files
- ROMs
- PUP Media files
- Any sort of licensing text that goes against the licensing of this repository

By submitting this PR, you are agreeing that any artwork submitted is your own, or you have the legal right to distribute such artwork.

### CODE REVIEW

File: external/vpx-flintstones/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+DirectB2S
+
+Authors: [Hauntfreaks](https://vpuniverse.com/profile/5216-hauntfreaks/)  
+Version: 2.0  
+Download: [VP Universe](https://vpuniverse.com/files/file/12799-the-flintstones-williams-1994-b2s-full-dmd/)
+
+## Status 
+
+Minimum VPX Standalone build: 10.8.0-1983-b84441e
```
Comment (n-i-x) [inline external/vpx-flintstones/README.md:17]: should be 10.8.0-1989-a764013

This should match the bundle version you tested the table on

File: external/vpx-flintstones/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+| Playfield | Controls | Backglass | DMD | FPS | 
+|-----------|----------|-----------|-----|----------|
+| :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | 30 |
```
Comment (n-i-x) [inline external/vpx-flintstones/README.md:20]: copy this data to the top-level README.md and create a link to this table

File: external/vpx-flintstones/TheFlintstones(Williams1994)v1.26.ini [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ViewCabScaleZ = 1.099294
+ViewCabRotation = 90.000000
+ViewCabPlayerX = -48.500000
+ViewCabPlayerY = -0.000000
+ViewCabPlayerZ = -303.220062
+ViewCabLookAt = 5.000000
+ViewCabFOV = 52.879169
+ViewCabLayback = 48.000000
+
+[Standalone]
```
Comment (n-i-x) [inline external/vpx-flintstones/TheFlintstones(Williams1994)v1.26.ini:17]: these look like the DMD settings.

PinMAMEWindowX, PinMAMEWindowY should probably be in VPinballX.ini if they aren't already. We only want override settings in the per-table configs.

**Review by n-i-x — CHANGES_REQUESTED** [review]
Nice first attempt! A few nits.

File: external/vpx-flintstones/TheFlintstones(Williams1994)v1.26.ini (hunk 3 above)
Comment (Bla1ze) [reply external/vpx-flintstones/TheFlintstones(Williams1994)v1.26.ini:17]: Those are table positions to create a better POV. Not related to the DMD. I'll double check though as they may not be needed.

Comment (Bla1ze) [reply external/vpx-flintstones/TheFlintstones(Williams1994)v1.26.ini:17]: It was just adjustments to the POV, it cleaned up black empty space at the top of the table. It's an adjustment users can make themselves though so fine to leave it out.

Comment (n-i-x) [reply external/vpx-flintstones/TheFlintstones(Williams1994)v1.26.ini:15]: No.. just wanted to make sure this wasn't something that should be applied everywhere. That should move into the main .ini.

**Review by n-i-x — APPROVED** [review]

File: external/vpx-flintstones/TheFlintstones(Williams1994)v1.26.ini (hunk 3 above)
Comment (Bla1ze) [reply external/vpx-flintstones/TheFlintstones(Williams1994)v1.26.ini:15]: No. It should remain a per table setting.

**Pull request merged**